controllers/nodeupgrade_controller.go

```diff
@@ -168,14 +167,23 @@ func (r *NodeUpgradeReconciler) reconcile(ctx context.Context, log logr.Logger,
 		return ctrl.Result{}, nil
 	}
 
+	configMap := &corev1.ConfigMap{}
+	if err := remoteClient.Get(ctx, types.NamespacedName{Name: constants.UpgraderConfigMapName, Namespace: constants.EksaSystemNamespace}, configMap); err != nil {
+		return ctrl.Result{}, err
+	}
+	if configMap.Data == nil {
+		return ctrl.Result{}, errors.New("upgrader config map is empty")
+	}
+	upgraderImage := configMap.Data[nodeUpgrade.Spec.KubernetesVersion]
```

Comment:
i meant something like 
```suggestion
	upgraderImage, ok := configMap.Data[nodeUpgrade.Spec.KubernetesVersion]
	if !ok {
	    return ctrl.Result{}, fmt.Errorf("upgrader image corresponding to EKS Distro version %s not found in the configmp", nodeUpgrade.Spec.KubernetesVersion)
	}
```

Reply:
Ah yes. I totally agree. Thanks!